src/Libraries/CoreNodes/Math.cs

```diff
@@ -17,7 +17,7 @@ namespace DSCore
     public static class Math
     {
         /// <summary>
-        ///     Generates a random double in the range of [0, 1).
+        ///     Generates a random double in the range of (0, 1).
```

Comment:
Actually, `[` was right here because according to the docs the value can be 0: https://docs.microsoft.com/en-us/dotnet/api/system.random.nextdouble?view=net-5.0

Reply:
@mmisol We did see this doc but from our own testing with millions of numbers the 0 was never hit.

Reply:
@QilongTang The value space is huge. I can say for sure you didn't get all possible values 😄

Reply:
After numerous testing, @mjkkirschner and I decided to update the range here as majority of our users would benefit from this..

Reply:
I don't see how giving misleading documentation could benefit users.

Reply:
@mmisol you could be right of course - I am not a statistician -  my hypothesis was that we should see a 0 once or twice in a million results - but... I guess I am super wrong:

https://stackoverflow.com/questions/58241868/can-random-uniform0-1-ever-generate-0-or-1

the response here is a few quadrillion... I'm not so sure about that either...

I think the result of system.random and our own method is confused by the fact that we do some scaling - so it's not clear if we actually have returned 0 and the result is loosing precision.